cmd/katib-controller/v1beta1/main.go

```diff
@@ -110,7 +111,9 @@ func main() {
 
 	// Create a new katib controller to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
-		MetricsBindAddress:     initConfig.ControllerConfig.MetricsAddr,
+		Metrics: metricsserver.Options{
+			BindAddress: initConfig.ControllerConfig.MetricsAddr,
+		},
```

Comment:
I addressed this breaking: https://github.com/kubernetes-sigs/controller-runtime/pull/2407